File: FirebaseApp (Healthia)/app/src/main/java/com/example/firebaseapp/notifications/APIService.java
package com.example.firebaseapp.notifications;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.Headers;
import retrofit2.http.POST;

public interface APIService {
    @Headers({
            "Content-Type:application/json",
            "Authorization:key=AAAAfv6YjN8:APA91bEnHW7iY-8uC2a0-jlpT9DJCLi9ht0fgGX_y67BgRHQFw-PUb3mh5LbzEkBZ3KeOfKZGAy-3yvM--AHgE6G6MjnouICX0otXtR697NcJ1NsWSQALqRx6yxW72c0xPBLbqPWfoVF"
    })

    @POST("fcm/send")
    Call<Response> sendNotification(@Body Sender body);
}

File: Healthia/app/src/main/java/com/example/healthia/notifications/APIService.kt
package com.example.healthia.notifications

import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.Headers
import retrofit2.http.POST

interface APIService {
    @Headers("Content-Type:application/json", "Authorization:key=AAAA3sSgh4A:APA91bEBCSPia0XOhDf-diVxVo6HAppvrilq8reShdQ-IeToy2v6by4jq5XZCpdKnD2-PnldH-B-Gu4YML-T_faL5UeFl_HNgrgB6kL4R9SyQoepvtuGYpD_kvA0734Me0KFW2eOcVHC")
    @POST("fcm/send")
    fun sendNotification(@Body body: Sender?): Call<Response?>
}